bigml/ensemble.php

```diff
@@ -132,6 +140,50 @@ public function __construct($ensemble, $api=null, $max_models=null, $storage="st
          $this->multi_model = new MultiModel($models, $this->api);
       }
 
+      if ($this->distributions == null) {
+          try {
+              $this->distributions = array();
+              foreach ($models as $model) {
+                  $distributions = $this->distributions;
+                  $distributions[] = array('training'=>array('categories'=> $model->tree->distribution));
+              }
+          }
+          catch(Exception $e) {
+              $this->distributions = array();
+              foreach ($models as $model) {
+                  $distributions = $this->distributions;
+                  $distributions[] = $model->object->model->distribution;
+              }
+          }
+      }
+
+      if (!$this->regression) {
```

Comment:
Good catch. The php bindings don't generally support boosting, but I've added the "non-boosted" check like the python bindings use.